packages/@react-aria/ssr/src/SSRProvider.tsx

```diff
@@ -75,7 +76,7 @@ export function useSSRSafeId(defaultId?: string): string {
     console.warn('When server rendering, you must wrap your application in an <SSRProvider> to ensure consistent ids are generated between the client and server.');
   }
 
-  return useMemo(() => defaultId || `react-aria-${ctx.prefix}-${++ctx.current}`, [defaultId]);
+  return useMemo(() => defaultId || `react-aria${ctx.prefix}-${++ctx.current}`, [defaultId]);
```

Comment:
Maybe we could keep `'0'` as the default prefix here for backward compatibility? Users might have existing snapshot tests that rely on this for example.

```suggestion
  return useMemo(() => defaultId || `react-aria${ctx.prefix || '-0'}-${++ctx.current}`, [defaultId]);
```

Reply:
I think with your solution, the generated IDs might be non-sequential in a surprising way. In the test on this PR, I think we'd get these IDs:

```
        "react-aria-0-1",
        "react-aria-2-1",
        "react-aria-2-2-1",
        "react-aria-0-3",
        "react-aria-4-1",
```

Alternatively, I could start the prefix at `0`, giving:

```
        "react-aria-0-1",
        "react-aria-0-2-1",
        "react-aria-0-2-2-1",
        "react-aria-0-3",
        "react-aria-0-4-1",
```

Giving backwards compatibility for some sequences. The downside I see is that it's a waste of a couple characters since they'll always start with `0-`. What do you think?

Reply:
Yeah it's probably fine to change them. We can treat it like any other DOM change we might make, which also might break snapshots. 👍